pangeo_forge_recipes/executors/prefect.py

```diff
@@ -43,3 +44,22 @@ def compile(pipeline: Pipeline):
     @staticmethod
     def execute(flow: Flow):
         return flow.run()
+
+
+@task
+def run_pipeline_with_dask(pipeline):
+    delayed = DaskPipelineExecutor.compile(pipeline)
+    # how do we make sure this uses the right dask cluster?
+    delayed.compute()
```

Comment:
Can we get the cluster client in scope here? Dask will default to the "current" distributed client, the most recently created one.

Reply:
`distributed.client._global_clients` is a weak dict containing all the clients that have been instantiated. We could ensure that there is exactly one that is still active. I'm no *too* worried about it, though, since there is only one cluster, and the recipes themselves should not be creating any.